### DISCUSSION TITLE
增加NotionAI源

### DISCUSSION THREAD

**Discussion by lizhemin15** [immersive-translate/immersive-translate#424] [Ideas]
### 您希望的更新和改进是什么 | Update or Improve

您好，建议加入NotionAI源，目前这是国内用户快速、低成本、无限量使用GPT3.5的一个非常好的路径，只要有NotionAI就可以无限使用，没有TOKEN限制。而NotionAI能够以非常低的价格得到。

### 补充说明 | Additional context

目前已经有人把NotionAI很好的用起来了，我认为这几种方式都很值得借鉴，对于愿意折腾的人可以自己获取token访问，对于不想折腾的人提供了一个简单的填写key的方式。作者也开源了源代码，并且开放了API。
https://getquicker.net/Sharedaction?code=41345e7b-0427-48ef-fb90-08db350ccd85

此外，基于作者的API，也有人接入了类似的翻译软件，效果非常好
https://getquicker.net/Sharedaction?code=04393db9-f4bc-4871-7fb6-08db2506d1ed

但是以上都局限于Quicker本身的平台，所以您如果也能接入，结合您跨平台的特性，我想很多人能够用上的话应该会效果非常好。

**Comment by lizhemin15** [comment]
![image](https://user-images.githubusercontent.com/57975421/235278493-ad3ec5f2-8c5b-4dc3-83a6-f43731d11f5e.png)
这是基于NotionAI的效果，非常炸裂，可以给出更多有意义的信息。

**Comment by versun** [comment]
之后会考虑,但优先级不会太高哈, 因为要用notionAI的话,要操作cookie, 涉及到隐私安全问题
